### PR TITLE
Pbi 322280 322532 boolean alt display name

### PR DESCRIPTION
**Pull request by jasnames**

### CODE REVIEW

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
+This table lists what AF objects can and cannot be included in a PI to AVEVA Data Hub data transfer.
 
-| AF Object                                                    | Included in a PI to OCS transfer?      |
+| AF Object                                                    | Included in a PI to AVEVA Data Hub transfer?      |
 | ------------------------------------------------------------ | -------------------------------------- |
 | Elements                                                     | Yes                                    |
 | Attributes                                                   | Yes                                    |
 | PI point references                                          | Yes                                    |
-| Constant values/string                                       | Yes                                    |
+| Constant values/string                                       | Yes*                                   |
```
Comment (osisoft-mbishop) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md:23]: Is this asterick supposed to connect to the note? If so, it's not immediatly apparent. I suggest swapping the asterick and **Note:** our for a superscript 1 (`<sup>1</sup>`)

Comment (jasnames) [reply content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md]: Sure, I'm fine with it that way. Change made.

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md [hunk 2]
```diff
@@ -2,23 +2,25 @@
...
+
+You can transfer the following AF element data into AVEVA Data Hub:
 
 * Simple PI point attributes
 * Static attribute values
 
 A simple PI point attribute consists of only a server and tag name. This data does not include any data retrieval qualifiers. The associated event data contains simple PI point attributes with the same historical and streaming transfer mechanism as points not associated with AF. 
 
-This table lists what AF objects can and cannot be included in a PI to OCS data transfer.
+This table lists what AF objects can and cannot be included in a PI to AVEVA Data Hub data transfer.
```
Comment (osisoft-mbishop) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md:16]: I think this Data Hub branding is intentional on your part, but just want to flag it to make sure.

**Review by osisoft-mbishop — DISMISSED** [review]
Looks good, although see my comments on branding--just making sure that it's intended.

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md (hunk 2 above)
Comment (jasnames) [reply content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/af-data-transferred.md:16]: Yep, all good. This feature is in ADH not OCS, but the TPM wants to make sure they are aware of it.

**Review by osisoft-mbishop — APPROVED** [review]

**Pull request merged**